Watchman.Discord/Areas/Users/Services/WarnsService.cs

```diff
@@ -62,14 +63,15 @@ public int GetWarnsCount(ulong userId, ulong serverId, DateTime from)
 
         public IEnumerable<WarnEvent> GetWarnEvents(ulong serverId, ulong receiverId, DateTime from =new DateTime())
         {
-            var query = new GetWarnEventsQuery(serverId, receiverId, from, to: DateTime.Now);
+            var query = new GetWarnEventsQuery(serverId, receiverId, new TimeRange(from, DateTime.Now));
             var response = this._queryBus.Execute(query);
             return response.WarnEvents;
         }
 
         private IEnumerable<KeyValuePair<string, string>> WarnEventsToKeyValue(IEnumerable<WarnEvent> warns, ulong mentionedUserId)
         {
             var warnEventPairs = new List<KeyValuePair<string, string>>();
+            var warnss = warns.ToList();
```

Comment:
to nic nie robi
ToList tutaj chyba nie jest potrzebny

Reply:
debugowałem bug z TimeRange.Contains() i zapomniałem usunąć